app/actions/storage/index.js

```diff
@@ -3,19 +3,22 @@
 
 import {AsyncStorage} from 'react-native';
 import {batchActions} from 'redux-batched-actions';
-import {GeneralTypes, TeamsTypes, UsersTypes} from 'service/constants';
-import Client from 'service/client';
+import {ChannelTypes, GeneralTypes, TeamsTypes, UsersTypes} from 'service/constants';
 
 export function loadStorage() {
     return async (dispatch, getState) => {
         try {
             const data = JSON.parse(await AsyncStorage.getItem('storage'));
-            const {token, url, currentTeamId} = data;
+
+            const {token, url, currentTeamId, ...otherStorage} = data;
             const credentials = {token, url};
 
+            const currentChannelId = otherStorage[currentTeamId] ? otherStorage[currentTeamId].currentChannelId : '';
```

Comment:
Yeah, actually I can't come up with any cases where the team id would clash with something else here. Great idea to handle anything team-specific this way!